include/librealsense2/h/rs_types.h

```diff
@@ -212,6 +213,8 @@ typedef enum rs2_matchers
 typedef struct rs2_device_info rs2_device_info;
 typedef struct rs2_device rs2_device;
 typedef struct rs2_error rs2_error;
+typedef struct rs2_string rs2_string;
```

Comment:
What is the purpose of rs2_string vs rs2_log_message?

Reply:
Removed rs2_string, as discussed